groups/font-features.yml

```diff
@@ -0,0 +1,3 @@
+# https://drafts.csswg.org/css-fonts-4/#font-rend-props
```

Comment:
We don't yet have a description field for groups, but probably will eventually. Would you mind writing one in a comment here? This name is going to be confusing since it's not clear how this is different from the parent's "Fonts" group (i.e., we're talking about features of fonts themselves, not web platform features relating to fonts).

Reply:
Added